common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockHandler.java

```diff
@@ -143,4 +179,26 @@ private void checkAuth(TransportClient client, String appId) {
     }
   }
 
+  /**
+   * A simple class to wrap all shuffle service wrapper metrics
+   */
+  private class ShuffleMetrics implements MetricSet {
+    private final Map<String, Metric> allMetrics;
+    private final Timer timeDelayForOpenBlockRequest = new Timer();
+    private final Timer timeDelayForRegisterExecutorRequest = new Timer();
+    private final Meter transferBlockRate = new Meter();
```

Comment:
Thank you for suggestions. That makes variable names much better. Already addressed it.